lib_version.py

```diff
@@ -0,0 +1,25 @@
+# BuildFox ninja generator
+
+import re
+
+MAJOR = 1
+MINOR = 0
+VERSION = "%d.%d" % (MAJOR, MINOR)
+
+# Simple major.minor matcher
+re_version = re.compile(r"^(\d+)\.(\d+)$")
+
+def parse(version):
```

Comment:
hey, no need to complicate stuff, here is one from [here](http://stackoverflow.com/questions/11887762/how-to-compare-version-style-strings) :

	def versiontuple(v):
		return tuple(map(int, (v.split("."))))

	versiontuple("2.3.1") > versiontuple("10.1.1")

And this function can be put in lib_util.py